stats/utils.py

```diff
@@ -0,0 +1,21 @@
+"""
+Misc utilities.
+"""
+
+
+def get_rowset_mapping(result_sets, column_names):
+    """
+    Returns a list of mapped fields to the passed headers.
+    """
+
+    headers = result_sets['headers']
+    return {column: headers.index(column.upper()) for column in column_names}
+
+
+def column_names_from_columns(db, table_name):
```

Comment:
Probably better renamed as `column_names_from_table`, from the outside, that's what's happening anyway. You are passing in a table, and getting columns names back.

Reply:
Fair, I can rename.